Add/terraform env creation

kalbir opened the pull request:
Add automatic environment creation to docs

@jojuolape [has
added](ministryofjustice/cloud-platform-environments#166)
functionality for us to automatically generate the namespace resource
files using terraform. He also did a draft of updating the guide to
reflect the change.

I've built on that to change the flow of the document a little and
remove some of the bits that were now overly wordly or incorrect. I've
tested out the method as well to make sure that this is resonably clear from a
user perspective.

Commit:
added terraform docs

kalbir committed:
Add automatic environment creation to docs

@johuolape [has
added](https://github.com/ministryofjustice/cloud-platform-environments/pull/166)
functionality for us to automatically generate the namespace resource
files using terraform. He also did a draft of updating the guide to
reflect the change.

I've built on that to change the flow of the document a little and
remove some of the bits that were now overly wordly or incorrect. I've
tested out the method as well to make sure that this is resonably clear from a
user perspective.

razvan-moj-zz commented on 01-getting-started/002-env-create.md:
````diff
@@ -62,41 +62,97 @@ The namespaces directory contains a directory for each of the clusters that you
+```
 
-To get started copy the example versions of these files from the [`namespace-resources`](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespace-resources) directory in the `cloud-platform-evironments` repo into your `<servicename-env>` directory.
+Our terraform module creates a namespace on live-0 by default but if you would like to deploy to another cluster you can use:
````
this might cause confusion: resources are not created directly in any cluster, just the definition written to local files which then need to be PRd

kalbir replied:
I'll make it "the files for a new namespace..."

kalbir committed:
Clear up that it is files that are created

This command doesn't create a namespace on the cluster, you'll need to get this merged into master on the cloud platform environments repo to actually create the namespace.
